src/main/java/com/google/sps/data/ChallengeData.java

```diff
@@ -17,15 +17,29 @@
     private static final String WASTE_CHALLENGE_0 = "Create your own compost";
     //private static final String WASTE_CHALLENGE_1 = "Avoid disposable goods";
 
-    //Challenge steps
-    private static final List GARDENING_STEPS_0 = Arrays.asList("step 1", "step 2", "step 3");
-    private static final List RECYCLE_STEPS_0 = Arrays.asList("step 1", "step 2", "step 3");
-    private static final List WASTE_STEPS_0 = Arrays.asList("step 1", "step 2", "step 3");
+    /* Note: step 1, step 2, step 3 are place holders */
+    private static final List GARDENING_STEPS_PAIR_0 =
```

Comment:
nit: Usually variable names don't need to convey their type (PAIR) unless it improves readability. Additionally, since these are lists, there's no need for the _0 suffix. `GARDENING_STEPS`, `RECYCLE_STEPS`, `WASTE_STEPS` are probably better var names

Reply:
I have suffix because I intend to add more challenges for each category in the a later PR.